src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/InviteOrganizationUsersCommand.cs

```diff
@@ -39,11 +40,22 @@ ISendOrganizationInvitesCommand sendOrganizationInvitesCommand
     public const string NoUsersToInvite = "No users to invite.";
     public const string InvalidResultType = "Invalid result type.";
 
+
     public async Task<CommandResult<ScimInviteOrganizationUsersResponse>> InviteScimOrganizationUserAsync(OrganizationUserSingleEmailInvite request)
     {
-        var hasSecretsManager = await paymentService.HasSecretsManagerStandalone(request.InviteOrganization);
+        var hasSecretsManagerStandalone = await paymentService.HasSecretsManagerStandalone(request.InviteOrganization);
+
+        var orgUsers = await organizationUserRepository.GetManyDetailsByOrganizationAsync(request.InviteOrganization.OrganizationId);
+
+        if (orgUsers.Any(existingUser =>
+                request.Email.Equals(existingUser.Email, StringComparison.InvariantCultureIgnoreCase) ||
+                request.ExternalId.Equals(existingUser.ExternalId, StringComparison.InvariantCultureIgnoreCase)))
```

Comment:
Do we want to allow for the linguistic adjustments of `InvariantCulture` when comparing emails? I would've expected a byte-for-byte comparison using `OrdinalIgnoreCase`.